go.mod

```diff
@@ -112,8 +112,8 @@ require (
 	github.com/hashicorp/vault-plugin-secrets-ad v0.13.1
 	github.com/hashicorp/vault-plugin-secrets-alicloud v0.12.0
 	github.com/hashicorp/vault-plugin-secrets-azure v0.13.0
-	github.com/hashicorp/vault-plugin-secrets-gcp v0.13.0
-	github.com/hashicorp/vault-plugin-secrets-gcpkms v0.12.0
+	github.com/hashicorp/vault-plugin-secrets-gcp v0.6.6-0.20220617175110-38223d8fabc9
```

Comment:
Are these plugins all being downgraded?

Reply:
Nope, they're not being downgraded (although it does appear this way). These are [pseudo-versions](https://go.dev/ref/mod#pseudo-versions) from `go get` of the latest in the main branch for each plugin.

Reply:
The thought here is to integrate the latest from each plugin sooner in the 1.12 release cycle. This gives us time to fix any issues that could come up with the google dependency update.

Reply:
Alternatively, we could create 1.12 release branches in each of these plugins to issue releases from earlier. This is something we usually do towards the end of the release cycle though.

Reply:
Any idea why the pseudo-version chooses the value of v0.6? My first guess was it used the Release version instead of Tags, but the only release version of gcp secrets is v0.8 so that theory didn't hold.

Reply:
Yep! The pseudo-version will be based on the latest tag on the branch your `go get` is targeted at. Taking GCP secrets as an example, the last tagged release on the main branch was [v0.6.5](https://github.com/hashicorp/vault-plugin-secrets-gcp/releases/tag/v0.6.5). You can see that the pseudo-version in this PR has an incremented patch to v0.6.6.

We used to tag the plugin releases directly from the main branch. We changed that process though and now only issue tagged releases of plugins directly from their release branches. That's why it appears as if it's a downgrade in this case.

Reply:
Outside of creating the plugin release branch (release/vault-1.12.x) early + tagging a release, I can't think of a different way to integrate plugin changes into Vault sooner in the release cycle. Historically, we've been okay with pulling in pseudo-versions of plugins to integrate with Vault earlier.

Reply:
Thanks for explaining, that makes sense since we don't tag main now. It's kind of a subtle distinction since the tags page doesn't make it immediately clear which branch a tag's on without going to the individual commit, so good to have learned. PR looks good to me!

Reply:
Ya.. It's not obvious. The tags page could be improved to expose more of that information (if you're listening GitHub :))